runtime/src/main/kotlin/konan/worker/Freezing.kt

```diff
@@ -26,7 +26,8 @@ public class FreezingException() : RuntimeException()
 /**
  * Exception thrown whenever we attempt to mutate frozen objects.
  */
-public class InvalidMutabilityException() : RuntimeException()
+public class InvalidMutabilityException(where: Any) :
+        RuntimeException("mutation attempt of frozen $where (hash is 0x${(where as Any).hashCode().toString(16)})")
```

Comment:
What's the purpose of casting `Any` to `Any`?

Reply:
Maybe none, I just didn't want virtual dispatch.